src/entities/TxFunctionInfo.ts

```diff
@@ -0,0 +1,14 @@
+import Field from "./Contracts/Field";
+import ITxFunctionInfo from "./ITxFunctionInfo";
+
+export default class TxFunctionInfo implements ITxFunctionInfo {
```

Comment:
This is an entity, not a component, so it should be called after what it represents, which is just a TxFunction

The TxFunctionInfo is the component that knows how to render this entity